packages/jest-haste-map/src/__tests__/index.test.js

```diff
@@ -8,6 +8,15 @@
 
 'use strict';
 
+const crypto = require('crypto');
+
+function mockHashContents(contents) {
```

Comment:
seems like const + arrow function allows you to oneline this guy.

Reply:
The linter doesn't allow this 😢

And to have it always as multiline, I prefer using a proper function.

Reply:
You mean the Jest hoist plugin? What about just inlining it in the mock below?